src/pages/ContentfulPage.vue

```diff
@@ -265,10 +265,10 @@ export default {
 	},
 	apollo: {
 		query: contentfulEntries,
-		preFetchVariables({ route, client }) {
+		preFetchVariables({ route }) {
 			return {
 				contentType: 'page',
-				contentKey: route?.meta?.contentfulPage(route, client)?.trim(),
+				contentKey: route?.meta?.contentfulPage(route, this.apollo)?.trim(),
```

Comment:
Do your changes below work without changes to this file?

Reply:
It was the main problem, that client is undefined for some time.

Reply:
ok that's interesting

Reply:
`this` is also undefined in preFetchVariables during the prefetch process. Client needs to be passed in to the preFetchVariables function for this to render correctly on the server-side. I'm testing a fix for that now.